nilearn/plotting/img_plotting.py

```diff
@@ -672,26 +672,29 @@ def plot_prob_atlas(maps_img, anat_img=MNI152TEMPLATE, view_type='auto',
             If nothing is specified, the MNI152 template will be used.
             To turn off background image, just pass "anat_img=False".
         view_type : {'auto', 'contours', 'filled_contours', 'continuous'}, optional
-            By default view_type == 'auto', which means maps are overlayed as
-            contours if number of maps to display are more or
-            overlayed as continuous colors if number of maps are less.
+            By default view_type == 'auto', means maps will be displayed
+            automatically using any one of the three view types. The automatic
+            selection of view type depends on the total number of maps.
             If view_type == 'contours', maps are overlayed as contours
             If view_type == 'filled_contours', maps are overlayed as contours
             along with color fillings inside the contours.
             If view_type == 'continuous', maps are overlayed as continous
             colors irrespective of the number maps.
-        threshold : None, a str or a number, list of either str or number, optional
-            If threshold is a string it must finish with a percent sign,
-            e.g. "25.3%", and it is a percentile. Or if it is a number,
-            it should be a real number, in which case it is the value
-            to threshold at.
-            This option is served for two purposes, for contours and contour
-            fillings threshold serves to select the level of the maps
-            to display and same threshold is applied for color fillings.
-            For continuous overlays this threshold value serves to select
-            the maps which are greater than a given value or list of given
-            values. If None is given, the maps are thresholded with default
-            value.
+        threshold : 'auto', None, a str or a number, str or numbers in list
+            Optional parameter is used to threshold the maps image and the
+            values which lies above the threshold level will be visualized.
+            By default 'auto', maps are thresholded automatically by finding
+            a value with the input maps data.
```

Comment:
I agree that the strategy is hard to describe. Instead of describing it, I think that you should at least state what is the purpose of the "auto" keyword:

The default strategy, corresponding to "auto", computes a threshold that seeks to minimize (yet not eliminate completely) the overlap between maps for a better display.